JavaFXUserInterface/src/main/java/external/AssetDAOImpl.java

```diff
@@ -57,7 +61,7 @@ public void deleteAssetByID(int assetID) {
                 assets.add(createAssetFromQueryResult(rs));
             }
         } catch (SQLException e) {
-            e.printStackTrace();
+            logger.error("Exception in getAllLiveAssets(): ", e);
```

Comment:
To target the duplication issues: This method can be removed (both from the implementation java file and interface file) as it is never used in the UI module (aside from a test that can be deleted as well).

Reply:
The same comment for the createAssetFromQueryResult() method.

Reply:
I'm not sure to understand your suggestion here. Please reach out to me for this, I can include the change in the next commit.

Reply:
I pushed a commit to resolve this